tests/auto/client/test_contactsimporter.cpp

```diff
@@ -125,6 +125,50 @@ private Q_SLOTS:
         QCOMPARE(account.billingAddressState(), QString());
     }
 
+    void testImportingAccountWithoutCountry()
+    {
+        ContactsImporter importer;
+        QTemporaryFile file;
+        writeTempFile(file,
+        "David,Faure,Mr,+331454532 45,david.faure@kdab.com,KDAB,\"32, street name\",,,,,FR 12345");
+        QVERIFY(importer.importFile(file.fileName()));
+        const QVector<ContactsSet> contacts = importer.contacts();
+        QCOMPARE(contacts.size(), 1);
+        const SugarAccount account = contacts.at(0).account;
+        QCOMPARE(account.name(), QString("KDAB"));
+        // auto fill country based on phone number
+        QCOMPARE(account.billingAddressCountry(), QString::fromUtf8("France"));
+    }
+
+    void testImportingAccountWithDiffLocale()
```

Comment:
a data-driven unittest would be really appropriate here, I think.

Reply:
fixed